app/assets/javascripts/L.GeoJSONvtTiles.js

```diff
@@ -267,6 +267,8 @@ L.GeoJsonvtTiles =  L.TileLayer.Canvas.extend({
             style: {color:'yellow'}
           });
           if (data && data.features && data.features.length) {
+            self.prepareFeatureData(data);
+
```

Comment:
It's be better to use passed callback to the constructor, see (options params)[
https://github.com/ondrej-hosak/las_control/blob/0900da4d826d283e05d1646c46058e62ae2b5fef/app/assets/javascripts/L.GeoJSONvtTiles.js#L23)